router/core/router.go

```diff
@@ -924,9 +926,21 @@ func (r *Router) newServer(ctx context.Context, routerConfig *nodev1.RouterConfi
 			h.ServeHTTP(w, r)
 		})
 	})
+
+	// Adds Brotli compressor
+	brCompressor := middleware.NewCompressor(5)
```

Comment:
Please ensure that also GraphQL specific content types are added to the list of supported compression content types https://github.com/go-chi/chi/blob/ef31c0bff3f8061e6fba7085691e4970a3d1b7da/middleware/compress.go#L17